test/specs/scenarios/crossDevice.js

```diff
@@ -215,6 +211,7 @@ export const crossDeviceScenarios = async (lang) => {
 
       it('should send sms and navigate to "Check your mobile" screen ', async () => {
         driver.get(baseUrl)
+        driver.navigate().refresh()
```

Comment:
Why is this browser refresh needed here in this test?

Reply:
For some reason, on safari, I had noticed if this was not there, it would never pass.